rfcs/0005-fork-selection.md

```diff
@@ -0,0 +1,166 @@
+# Fork Selection
+ This RFC defines Solana's *Nakomoto Fork Selection* algorithm based on time locks. It satisfies the following properties:
+
+
+* A voter can eventually recover from voting on a fork that doesn't become the fork with the desired finality.
```

Comment:
It bugs me that in this first bullet, you say "desired finality." That doesn't really mean anything to me. Biggest question is, how are you defining finality? Shouldn't that be explained first? Second, what makes finality "desirable"?